### PR TITLE
[HOTFIX] Addressed Uniqueness bucket issue

### PR DESCRIPTION
**Pull request by ashishYugen**
### Description

Addressed Uniqueness bucket issue. For more context [refers](https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucketnamingrules.html)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -384,8 +384,20 @@ terraform apply -no-color -auto-approve --var-file=../../canso-dataplane-configs
...
+
+```console
+canso-{UNIQUE_TENANT_NAME}-materialised-features
+canso-{UNIQUE_TENANT_NAME}-materialised-logs
+canso-{UNIQUE_TENANT_NAME}-materialised-artifacts
+```
+
+> [!IMPORTANT]
+> Dependencies:
+> Ensure you replace UNIQUE_TENANT_NAME with your actual value in the
```
Comment (soumanta-yugen) [inline README.md:399]:
```suggestion
> Ensure you replace `UNIQUE_TENANT_NAME` with your actual value in the
```

**Review by soumanta-yugen — COMMENTED** [review]
One minor nitpick, rest looks good. 

@mahantesh-yugen Feel free to take a look and share your feedback.

**Comment by mahantesh-yugen** [comment]
We are asking the user to give the same input in 3 different places. Can we prevent this?

One Idea is to create a variable "tenant_name" and then use it like `var.tenant_name + each.value` [here](https://github.com/Yugen-ai/canso-data-plane-k8s-cluster-tf/blob/main/modules/canso-s3/main.tf#L4).

This will also allow the user to pass the tenant_name in the command instead of updating the var file.
Eg. `terraform apply -no-color -auto-approve --var-file=../../canso-dataplane-configs/s3/auto.tfvars -var tenant_name="NAME"`

**Comment by ashishYugen** [comment]
> We are asking the user to give the same input in 3 different places. Can we prevent this?
> 
> One Idea is to create a variable "tenant_name" and then use it like `var.tenant_name + each.value` [here](https://github.com/Yugen-ai/canso-data-plane-k8s-cluster-tf/blob/main/modules/canso-s3/main.tf#L4).
> 
> This will also allow the user to pass the tenant_name in the command instead of updating the var file. Eg. `terraform apply -no-color -auto-approve --var-file=../../canso-dataplane-configs/s3/auto.tfvars -var tenant_name="NAME"`

@mahantesh-yugen In this scenario, the bucket will be created with UNIQUE_TENANT_NAME as a prefix. Could you run a local POC to confirm that it's working correctly?

**Comment by mahantesh-yugen** [comment]
> > We are asking the user to give the same input in 3 different places. Can we prevent this?
> > One Idea is to create a variable "tenant_name" and then use it like `var.tenant_name + each.value` [here](https://github.com/Yugen-ai/canso-data-plane-k8s-cluster-tf/blob/main/modules/canso-s3/main.tf#L4).
> > This will also allow the user to pass the tenant_name in the command instead of updating the var file. Eg. `terraform apply -no-color -auto-approve --var-file=../../canso-dataplane-configs/s3/auto.tfvars -var tenant_name="NAME"`
> 
> @mahantesh-yugen In this scenario, the bucket will be created with UNIQUE_TENANT_NAME as a prefix. Could you run a local POC to confirm that it's working correctly?

@ashishYugen Tested in local, working correctly

**Review by soumanta-yugen — APPROVED** [review]

**Pull request merged**